examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/VideoPlayer.java

```diff
@@ -0,0 +1,17 @@
+/** a.k.a. TV */
+public class VideoPlayer {
+  private long nodeID;
+  private byte fabricIndex;
+
+  private boolean isConnected;
+  // Empty if we don't have a connection to the TargetVideoPlayer yet (for e.g. when the casting app
```

Comment:
Or if this is not a casting video player.

Reply:
Currently, provided we're connected to the video player, we do return a list of all the endpointIds with all their clusterIds. How is a "casting video player" defined anyway - are we supposed to check for specific clusters on the endpoints it hosts?

Reply:
device type of the video player will be different.